src/app/f2f/utils.js

```diff
@@ -21,4 +22,16 @@ function formatDate(date, format) {
   }
 }
 
-module.exports = { formatDate };
\ No newline at end of file
+function beforeNow(_value, timePeriod, timeUnit) {
+  let dateFormat = "YYYY-MM-DD";
+  let test = moment(_value, dateFormat);
```

Comment:
This is interesting so we are still comparing against "now" but we're using moment. Do you know the mechanics of how this is different to what we were doing before?

Reply:
https://gds.slack.com/archives/C03H1GP6NTA/p1689846860615609